filters/builtin/compress.go

```diff
@@ -268,9 +272,20 @@ func unsupported() {
 
 func newEncoder(enc string, level int) (encoder, error) {
 	switch enc {
+	case "br":
+		if level > brotli.BestCompression {
```

Comment:
this check you already did on filter creation (CreateFilter())

Reply:
removed :)